config/tailwind.config.js

```diff
@@ -10,7 +10,8 @@ module.exports = {
   theme: {
     extend: {
       fontFamily: {
-        sans: ['Roboto', ...defaultTheme.fontFamily.sans]
+        sans: ['Roboto', ...defaultTheme.fontFamily.sans],
+        karla: ['Karla', ...defaultTheme.fontFamily.sans]
```

Comment:
are we only using the `karla` font in the `about` section?

Reply:
yes